### PR TITLE
Feature: buttons group component

### PR DESCRIPTION
**Pull request by Aparacki**
DEFAULT:
image

EQUAL:
image

FULL:
image

### CODE REVIEW

File: packages/button-group/src/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import {Button} from '@smashing/button'
+import styled from 'styled-components'
+import {getButtonStyle} from '@smashing/button/lib/esm/styles'
```
Comment (Aparacki) [inline packages/button-group/src/styles.tsx]: I'm not sure about imports with this path

**Comment by curryy** [comment]
Maybe this component should be part of `radio` component? It looks different but does the same thing.

File: stories/button-group.stories.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    value: 3
+  },
+  {
+    label: 'and typesetting industry',
+    value: 4
+  }
+]
+addDecorator(withA11y)
+
+storiesOf('Atomic Elements|Buttons Group', module)
```
Comment (hzub) [inline stories/button-group.stories.js]: I'd move it from `Atomic Elements` to `Form`

**Comment by Idered** [comment]
@curryy @Aparacki  I would rename it to `radio-group` and add prop `appearance: 'button' | 'radio'`. Do we want to do it in this PR or merge this as is? I would prefer to do it in this PR

**Comment by hzub** [comment]
@Idered About `radio` appearance - how it should look like? Just a vertical list of radio buttons?

**Comment by Idered** [comment]
@Aparacki @hzub @curryy let's have a short meeting about refactoring this to radio-group.

**Comment by hzub** [comment]
@Aparacki @curryy @Idered 
right now there is a proposal to change everything to `ControlGroup` with different appearances: `ButtonGroup`, `RadioGroupHorizontal`, `RadioGroupVertical` (default) and possibility of multi-selection
if everybody's OK with this, we can continue with the implementation

**Comment by Idered** [comment]
@hzub 
```tsx
/**
 * Appearance applied directly to each group item
 */
controlAppearance: ButtonAppearanceType | RadioAppearanceType | CheckboxAppearanceType
/**
 * Appearance of group
 */
groupAppearance: 'button' | 'radio-horizontal' | 'radio-vertical' | 'checkbox-vertical | 'checkbox-horizontal'
```

We should keep appearance value lower case.

**Comment by hzub** [comment]
@Idered 
Using described appearance property set (controlAppearance + groupAppearance), which one of them would be responsible for changing group mechanics (multi/single select)?

**Comment by Idered** [comment]
@hzub If value is array then multiple controls can be selected - each control can be toggled. We can also add isMultiSelect like in select-menu I guess but I don't see it as necessary

**Comment by hzub** [comment]
@Idered 👌 
can you provide example scenarios of mixing `controlAppearance` and `groupAppearance`? I guess there is some redundancy after all

**Comment by Idered** [comment]
```tsx
<ControlGroup
  value={[]}
  items={items}
  controlAppearance="flat" // <Button appearance="outline" />
  groupAppearance="button"
/>
<ControlGroup
  value={''}
  items={items}
  controlAppearance="outline" // <Radio appearance="outline" />
  groupAppearance="radio-vertical"
/>
```

**Comment by hzub** [comment]
👌

**Comment by Idered** [comment]
This can be closed in favor of #96 right?

**Pull request closed without merging**